lib/app/modules/addOrUpdateAlarm/views/add_or_update_alarm_view.dart

```diff
@@ -122,633 +125,717 @@ class AddOrUpdateAlarmView extends GetView<AddOrUpdateAlarmController> {
+                  if (res){
+                    Get.defaultDialog(
+                      titlePadding: const EdgeInsets.symmetric(
+                        vertical: 20,
                       ),
-                      onPressed: () async {
-                        Utils.hapticFeedback();
-                        if (controller.userModel != null) {
-                          controller.offsetDetails[controller.userModel!.id] = {
-                            'offsettedTime': Utils.timeOfDayToString(
-                              TimeOfDay.fromDateTime(
-                                Utils.calculateOffsetAlarmTime(
-                                  controller.selectedTime.value,
-                                  controller.isOffsetBefore.value,
-                                  controller.offsetDuration.value,
-                                ),
-                              ),
-                            ),
-                            'offsetDuration': controller.offsetDuration.value,
-                            'isOffsetBefore': controller.isOffsetBefore.value,
-                          };
-                        } else {
-                          controller.offsetDetails.value = {};
-                        }
-                        AlarmModel alarmRecord = AlarmModel(
-                          snoozeDuration: controller.snoozeDuration.value,
-                          offsetDetails: controller.offsetDetails,
-                          label: controller.label.value,
-                          isOneTime: controller.isOneTime.value,
-                          lastEditedUserId: controller.lastEditedUserId,
-                          mutexLock: controller.mutexLock.value,
-                          alarmID: controller.alarmID,
-                          ownerId: controller.ownerId,
-                          ownerName: controller.ownerName,
-                          activityInterval:
-                              controller.activityInterval.value * 60000,
-                          days: controller.repeatDays.toList(),
-                          alarmTime: Utils.timeOfDayToString(
-                            TimeOfDay.fromDateTime(
-                              controller.selectedTime.value,
-                            ),
-                          ),
-                          mainAlarmTime: Utils.timeOfDayToString(
-                            TimeOfDay.fromDateTime(
-                              controller.selectedTime.value,
-                            ),
-                          ),
-                          intervalToAlarm: Utils.getMillisecondsToAlarm(
-                            DateTime.now(),
-                            controller.selectedTime.value,
+                      backgroundColor: themeController.isLightMode.value
+                          ? kLightSecondaryBackgroundColor
+                          : ksecondaryBackgroundColor,
+                      title: 'Alarm already set for $time',
```

Comment:
I think this dialog might be bad for ux, it should ideally ask the user if they want to overwrite the settings for the new alarm (considering they might want something else enabled).